www/modules/sidebar/controllers/sidebar.client.controller.es6

```diff
@@ -1,7 +1,15 @@
 class SidebarController {
-  constructor(Authentication) {
-    this.user = Authentication.user;
+  /*@ngInject*/
+  constructor(Authentication, $state) {
+    Object.assign(this, {Authentication, $state});
+    this.user = this.Authentication.user;
+  }
+  singOut (){
+    this.state.go('auth.signin');
+    this.user = undefined;
```

Comment:
Why we assign undefined and not null?
I think null is more suitable in this case.